packages/arb-validator/cmd/arb-validator/arb-validator.go

```diff
@@ -48,28 +50,36 @@ func main() {
 	flag.Parse()
 	switch os.Args[1] {
 	case "create":
-		if err := createRollupChain(); err != nil {
+		if _, err := createRollupChain(); err != nil {
 			log.Fatal(err)
 		}
 	case "validate":
-		if err := cmdhelper.ValidateRollupChain("arb-validator", createManager); err != nil {
+		if err := cmdhelper.ValidateRollupChain("arb-validator", createManager, ""); err != nil {
+			log.Fatal(err)
+		}
+	case "both":
+		addr, err := createRollupChain()
+		if err != nil {
+			log.Fatal(err)
+		}
+		if err := cmdhelper.ValidateRollupChain("arb-validator", createManager, addr); err != nil {
 			log.Fatal(err)
 		}
 	default:
 	}
 }
 
-func createRollupChain() error {
+func createRollupChain() (string, error) {
```

Comment:
This should return a `common.address` rather than a string

Reply:
fixed